src/components/DashboardPage/DashbaordPage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { v4 as uuidv4 } from 'uuid';
+import excelCommunicator from "../../communication/excelCommunicator";
+import { FullDataInstanceInfo, Table, TableDictionary } from "../../types/data";
+import Dashboard from "../Dashboard/Dashboard";
+import ManagerPanel from "../ManagerPanel/ManagerPanel";
+
+const DashbaordPage = () => {
+    const [tableDictionary, setTableDictionary] = useState<TableDictionary>({});
+    const [fullDataInstanceInfo, setFullDataInstanceInfo] = useState<FullDataInstanceInfo>({});
+
+    const addDataInstanceTable = async (table: Table, info: any) => {
+        const dataInstanceId = uuidv4();
```

Comment:
I am again asking if this is something that is supposed to be generated in the client, I feel like it should be generated in the server

Reply:
its changed in the excel branch